components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesEndpoint.java

```diff
@@ -25,37 +25,24 @@
 import org.apache.camel.CamelContext;
 import org.apache.camel.Route;
 import org.apache.camel.spring.boot.actuate.endpoint.CamelRoutesEndpoint.RouteEndpointInfo;
+import org.springframework.boot.actuate.endpoint.AbstractEndpoint;
 import org.springframework.boot.actuate.endpoint.Endpoint;
 
 /**
  * {@link Endpoint} to expose {@link org.apache.camel.Route} information.
  */
-public class CamelRoutesEndpoint implements Endpoint<List<RouteEndpointInfo>> {
+public class CamelRoutesEndpoint extends AbstractEndpoint<List<RouteEndpointInfo>> {
 
-    private static final String ENDPONT_ID = "camelroutes";
+    private static final String ENDPOINT_ID = "camelroutes";
 
     private CamelContext camelContext;
 
     public CamelRoutesEndpoint(CamelContext camelContext) {
+        super(ENDPOINT_ID);
```

Comment:
Thanks for pointing it out, but already fixed in the latest updates where I extend from AbstractEndpoint.